qa/rpc-tests/wallet_import_export.py

```diff
@@ -59,15 +78,15 @@ def run_test(self):
 def parse_wallet_file(dump_path):
     file_lines = open(dump_path, "r", encoding="utf8").readlines()
     # We expect information about the HDSeed and fingerpring in the header
-    assert_true("HDSeed" in file_lines[4], "Expected HDSeed")
-    assert_true("fingerprint" in file_lines[4], "Expected fingerprint")
-    seed_comment_line = file_lines[4][2:].split()  # ["HDSeed=...", "fingerprint=..."]
-    assert_true(seed_comment_line[0].split("=")[1] != seed_comment_line[1].split("=")[1], "The seed should not equal the fingerprint")
```

Comment:
Hmm, we're not checking the fingerprint doesn't equal the seed anymore. It would be nice to retain that consistency check against the recovery phrase.

Reply:
I thought that was a bit of an odd check; why might one expect them to ever be equal? There are an infinite number of such negative assertions one could make, but they don't cover actual risks?